ciongke/cmd/hook/main.go

```diff
@@ -59,12 +59,24 @@ var defaultJenkinsJobs = map[string][]JenkinsJob{
 			Context:   "Jenkins Federation GCE e2e",
 		},
 		{
+			Name:      "kubernetes-pull-build-test-gci-federation-e2e-gce",
+			Trigger:   regexp.MustCompile(`@k8s-bot federation (gce )?(e2e )?test this`),
```

Comment:
I would suggest having some way of triggering this separately from the standard cvm test. Maybe `@k8s-bot federation (gci )?(e2e )?test this`? Same for the second.